docs/docs/basic-usage.md

```diff
@@ -41,6 +41,24 @@ python = "*"
 pytest = "^3.4"
 ```
 
+### Initialising a pre-existing project
+
+Instead of creating a new project, Poetry can be used to 'initialise' a pre-populated
+directory. It expects a standard Python package structure to do this:
+
+```text
+pre-existing-project
+├── README.rst
+└── pre_existing_project
+    └── __init__.py
+```
+
+To initialise this project, and interactively create a `pyproject.toml` file:
+
+```bash
+poetry init pre-existing-project
```

Comment:
Thanks for the changes!

One more note: `poetry init` doesn't expects the folder name here. Just use `poetry init` within the root folder.

Reply:
Oops! Hopefully this is better now.